gammapy/estimators/flux_point.py

```diff
@@ -308,7 +277,9 @@ def to_table(self, sed_type="likelihood", format="gadf-sed", formatted=False):
             Flux points table
         """
         if format == "gadf-sed":
-            idx = (Ellipsis, 0, 0)
+            # TODO: what to do with GTI info?
```

Comment:
astropy.table has a member metadata ("meta: dict, optional -> Metadata associated with the table").
Why is it not possible to use that field?

Reply:
In principle this is possible, however the `GTI` information is often a complete table with start and stop times...